client/ayon_maya/plugins/load/load_ornatrix_alembic.py

```diff
@@ -0,0 +1,133 @@
+from maya import cmds
+
+from ayon_core.settings import get_project_settings
+from ayon_core.pipeline import get_representation_path
+from ayon_maya.api import plugin
+from ayon_maya.api.pipeline import containerise
+from ayon_maya.api.lib import maintained_selection, unique_namespace
+from ayon_maya.api.plugin import get_load_color_for_product_type
+from ayon_core.lib import EnumDef
+
+
+class OxAlembicLoader(plugin.Loader):
+    """Ornatrix Alembic Loader"""
+
+    product_types = {"oxcache"}
+    representations = {"abc"}
+
+    label = "Ornatrix Alembic Loader"
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    @classmethod
+    def get_options(cls, contexts):
+        return [
+            EnumDef(
+                "import_options",
+            label="Import Options for Ornatrix Cache",
+            items={
+                0: "Hair",
+                1: "Guide"
+            },
+            default=0,
+        )
+    ]
+
+    def load(self, context, name, namespace, options):
+        cmds.loadPlugin("Ornatrix", quiet=True)
+        product_type = context["product"]["productType"]
```

Comment:
Should we move this variable definition closer to where it's used - near line 71?